script/install.sh

```diff
@@ -126,6 +126,37 @@ linux() {
     cp "$HOME/.local/zed$suffix.app/share/applications/zed$suffix.desktop" "${desktop_file_path}"
     sed -i "s|Icon=zed|Icon=$HOME/.local/zed$suffix.app/share/icons/hicolor/512x512/apps/zed.png|g" "${desktop_file_path}"
     sed -i "s|Exec=zed|Exec=$HOME/.local/zed$suffix.app/bin/zed|g" "${desktop_file_path}"
+
+    # Copy PolicyKit if pkexec is available
+    if command -v pkexec >/dev/null 2>&1; then
+        echo "Zed requires sudo access to setup polkit policy file and elevation script."
```

Comment:
I think I'd prefer if we didn't handle this by exiting. Maybe change it to "Rerun this script with polkit so we can setup the policy file", or provide a separate script people can run like `sudo curl -f https://zed.dev/setup-polkit.sh | sh` . I think it's important that we offer a streamlined, non-sudo install path for our users by default.